addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.xml

```diff
@@ -6,7 +6,6 @@
     <t t-name="im_livechat.legacy.PublicLivechatWindow">
         <div class="o_thread_window o_in_home_menu"
             t-att-data-thread-id="widget.messaging.publicLivechatGlobal.publicLivechat.id"
```

Comment:
Shouldn't this one be channel-id rather than thread-id then? Is it actually used in any test?

Reply:
will just drop it, it's not used anyway